x-pack/plugin/ent-search/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/entsearch/20_engine_put.yml

```diff
@@ -45,6 +45,12 @@ setup:
 
   - match: { result: "created" }
 
+  - do:
+      indices.exists_alias:
```

Comment:
The test complains about:

`java.lang.IllegalStateException: Unable to find a matching path for api [indices.exists_alias]`

I think you need to provide the alias name as part of the path instead of the body:
```
      indices.exists_alias:
      name: engine-test-engine
```

See [this test](https://github.com/elastic/elasticsearch/blob/6339691fe3dd95a880b0457fb230b308119feb11/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/indices.exists_alias/10_basic.yml#L4) for reference

Reply:
Thanks, these tests are fixed!